### PR TITLE
:heart_eyes: invoice updating optimization, downloads only new orders

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: ncf_pos/static/src/js/models.js [hunk 1]
```diff
@@ -310,11 +310,14 @@ odoo.define('ncf_pos.models', function (require) {
         get_orders_from_server: function () {
             var self = this;
             var day_limit = this.config.order_loading_options === 'n_days' ? this.config.number_of_days : 0;
+            var max_id = Math.max(..._.map(this.db.pos_all_orders, function(o){
```
Comment (yelizariev) [inline ncf_pos/static/src/js/models.js]: Que?

Comment (yelizariev) [reply ncf_pos/static/src/js/models.js]: ok, it's ES6 feature. You must check browser support before using it

Comment (gustavovalverde) [reply ncf_pos/static/src/js/models.js]: So, will this work in Chrome, Opera and Firefox?

Comment (KolushovAlexandr) [reply ncf_pos/static/src/js/models.js]: according to this 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax
it doesnt work only in the Internet Explorer

**Comment by gustavovalverde** [comment]
@KolushovAlexandr please check the conflicts

**Comment by KolushovAlexandr** [comment]
@gustavovalverde Done

**Pull request merged**